### ISSUE TITLE
DescriptorMatcher only requires desc1, desc2, not LAFs

### ISSUE THREAD

**Issue by 007anupamdas** [kornia/kornia#2975]
## 📚 Documentation

The document clearly dictates that forward method of DescriptorMatcher requires 4 parameters, viz desc1, desc2, lafs1, lafs2. But when you go through codebase, it requires only the descriptors as input. Including LAFs as per the documentation, throws error. 
Kindly update.

**Comment by edgarriba** [comment]
@007anupamdas are you open to send a PR to fix the docs ?

**Comment by 007anupamdas** [comment]
Sorry for the delay, as I was involved in upcoming NISAR mission. I have altered many things in my code, had to comment a lot, there might be minor indentation issues. I am currently experimenting with SIFT, DEDODE, LOFT. My task is to find difference in two microwave images, which is very challenging.

[DPQED_lf.txt](https://github.com/user-attachments/files/19223550/DPQED_lf.txt)

**Comment by edgarriba** [comment]
@ducha-aiki maybe helpful here

**Comment by ducha-aiki** [comment]
@007anupamdas will fix, thank you

**Comment by edgarriba** [comment]
If anyone is interested in this, feel free to re-open the ticket. Thanks!

**Comment by ducha-aiki** [comment]
@ducha-aiki to do

**Comment by edgarriba** [comment]
@ducha-aiki are you planning to do it yourself ? otherwise add proper labeling in the ticket so that other contributors can pick it up and have enough information/context

**Comment by ducha-aiki** [comment]
Yes. To be more precise, only my work Cursor resets usage limits, I plan to ask it to check mismatch between docs and functions and fix it.
